helmfile/charts/prometheus-alerts/templates/alerts/disk-perf.yaml

```diff
@@ -0,0 +1,23 @@
+# Based on https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.diskPerf }}
```

Comment:
@crssnd this should allow disabling from config, right? (re comment on `files/disk-perf.yaml`)

Reply:
to be able to disable them from the overwrite configs you will need to add that option in this [file](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/prometheus-alerts-sc.yaml.gotmpl#L31) and also the value in [sc-config.yaml](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/sc-config.yaml)

Reply:
LGTM unless @crssnd has anything to add